index.html

```diff
@@ -1,279 +1,295 @@
+        <h2 class="header_title">I am a software developer</h2>
+        <p class="header_description">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to <strong>contact me.</strong>
         </p>
       </div>
-      <ul class="languages">
-        <li>Html</li>
-        <li>Css</li>
-        <li>Bootstrap</li>
-        <li>Ruby</li>
+      <ul class="social_media_link">
+        <li class="social_media_icon">
+          <a href=""><img src="./images/github.svg" alt="" /></a>
+        </li>
+        <li class="social_media_icon">
+          <a href=""><img src="./images/Linkedinicon.svg" alt="" /></a>
+        </li>
+        <li class="social_media_icon">
+          <a href=""><img src="./images/angellist.svg" alt="" /></a>
+        </li>
+        <li class="social_media_icon">
+          <a href=""><img src="./images/twitter.svg" alt="" /></a>
+        </li>
+        <li class="social_media_icon">
+          <a href=""></a><img src="./images/medium.svg" alt="" />
+        </li>
       </ul>
+    </section>
+
+    <section class="recent_works">
+        <div class="recent_work_header_row">
+            <h2 class="recent_works_header">My Recent Works</h2>
+            <img class="work_image_line" src="./images/Vector.svg" alt="image_line">
+        </div>
+
+
+        <div class="placeholder_container">
+            <div class="placeholder_div">
+                <img class="Placeholder_image" src="./images/ImgPlaceholder.png" alt="Placeholder_frame">
+
+                <div class="placeholder_description">
+                    <h3>Multi-Post Stories</h3>
+                    <p class="placeholder_text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+
+                    <div class="placeholder_language">
+                        <button class="placeholder_title text">css</button>
+                        <button class="placeholder_title text">html</button>
+                        <button class="placeholder_title text">javascript</button>
+                        <button class="placeholder_title text">Ruby</button>
+                    </div>
+                    <div class="see_project">
+                        <button class="see_project_button">See Project</button>
+                    </div>
+                </div>
+
+                
+            </div>
+            
+            <div class="recent_work_adjustment">
+                   <div class="recent_work">
+                    <div class="second_project first_project">
+                        <div class="first_project_holder">
+                           
+                            <button class="see_project_button2">See Project</button>
```

Comment:
[REQUIRED] your button during the hover and pressed state does not match the template as suggested by the previous reviewer

Reply:
@fobadara Thanks for your review, could you provide more clarity, I suppose you are referring to the button on the image below? If so, i have just fixed it.
![see_project_button](https://user-images.githubusercontent.com/60592651/194710945-7751d451-1c40-40d7-9860-78c9ea7020f4.jpg)

Reply:
Ensure you implement it on all buttons that are orange